localstack-core/localstack/services/s3/provider.py

```diff
@@ -581,14 +581,51 @@ def list_buckets(
         bucket_region: BucketRegion = None,
         **kwargs,
     ) -> ListBucketsOutput:
-        # TODO add support for max_buckets, continuation_token, prefix, and bucket_region
+        if continuation_token == "":
+            raise InvalidArgument(
+                "The continuation token provided is incorrect",
+                ArgumentName="continuation-token",
+            )
```

Comment:
it seems we do not have a snapshot test covering this, it would be great to add a snapshot validated test for it

Reply:
This is good, thanks for checking this! 👍